docs/release/release.md

```diff
@@ -31,8 +31,8 @@ For both, this is the complete list of valid variants:
 * _minor_
 * _patch_
 * _!pre \<pre-release-suffix\>_ (e.g. `!pre dev`)
-* _!pre\_major \<pre-release-suffix\>_ (e.g. `!pre_patch rc`)
-* _!pre\_minor \<pre-release-suffix\>_ (e.g. `!pre_patch beta`)
+* _!pre\_major \<pre-release-suffix\>_ (e.g. `!pre_major rc`)
+* _!pre\_minor \<pre-release-suffix\>_ (e.g. `!pre_minor beta`)
```

Comment:
@steveej These examples were probably copy-pasted and not modified. ; )

Reply:
thanks for catching!